theme/static/js/resourceLandingAjax.js

```diff
@@ -882,7 +899,7 @@ function filetype_keywords_update_ajax_submit() {
                     li.find('span').text(keywords[i]);
                     li.append('&nbsp;<a><span class="glyphicon glyphicon-remove-circle icon-remove"></span></a>');
                     $("#lst-tags-filetype").append(li);
-                    $(".icon-remove").click(onRemoveKeywordFileType);
+                    $("#lst-tags-filetype").find(".icon-remove").click(onRemoveKeywordFileType);
```

Comment:
@pkdash, this method can also be dynamically bound once.  https://stackoverflow.com/a/34896387

Reply:
The file type keywords are no available at the time of document load/ready. It is available only when the user selects to see metadata for a file type.